src/quo2/components/buttons/slide_button/style.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.buttons.slide-button.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.reanimated :as reanimated]))
+
+(defn slide-container
+  [slider-height slider-padding slider-width disabled]
+  {:background-color "#101B3A"
```

Comment:
@J-Son89 Thanks for the information. I tried using the colors from the quo2/foundations/colors. As per the UX, we needed an RGBA color, but when using the color, the content behind it (label in this case) will be partially visible because of the alpha channel. I have explained the issue in the PR notes as well.